samples/javascript_nodejs/53.teams-messaging-extensions-action-preview/bots/teamsMessagingExtensionsActionPreviewBot.js

```diff
@@ -53,8 +53,15 @@ class TeamsMessagingExtensionsActionPreviewBot extends TeamsActivityHandler {
 
         // This is a send so we are done and we will create the adaptive card editor.
         const adaptiveCard = AdaptiveCardHelper.createAdaptiveCardAttachment(submitData);
-        const responseActivity = { type: 'message', attachments: [adaptiveCard] };
-
+        var responseActivity = { type: 'message', attachments: [adaptiveCard] };
+        if(submitData.UserAttributionSelect=='true'){
+             responseActivity = { type: 'message', attachments: [adaptiveCard], channelData: {
+                onBehalfOf: [
+                    { itemId: 0, mentionType: 'person', mri: context.activity.from.id, displayname: context.activity.from.name }
+                ]
+            }}; 
+        }
```

Comment:
```suggestion
        if (submitData.UserAttributionSelect === 'true') {
             responseActivity = {
                 type: 'message',
                 attachments: [adaptiveCard],
                 channelData: {
                     onBehalfOf: [
                        {
                            itemId: 0,
                            mentionType: 'person',
                            mri: context.activity.from.id,
                            displayName: context.activity.from.name
                        }
                    ]
            }}; 
        }
```

Please confirm that changing `displayname` to `displayName` (camelCase) is appropriate. It appears to be  based on the JSON in [this doc](https://docs.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit?tabs=javascript%2Cjson-1), although not in the JS.

Reply:
either `displayname `or `displayName `bot are working fine

Reply:
I checked this one 
![image](https://user-images.githubusercontent.com/50989436/106560272-c7446400-654c-11eb-8bc6-73cd7f346ca6.png)